moveit_ros/planning_interface/test/CMakeLists.txt

```
@@ -13,6 +13,9 @@ if (CATKIN_ENABLE_TESTING)
   add_rostest_gtest(subframes_test subframes_test.test subframes_test.cpp)
   target_link_libraries(subframes_test moveit_move_group_interface ${catkin_LIBRARIES})
 
+  add_rostest_gtest(planning_scene_interface_clearScene_test planning_scene_interface_clearScene.test planning_scene_interface_clearScene.cpp)
+  target_link_libraries(planning_scene_interface_clearScene_test moveit_move_group_interface ${catkin_LIBRARIES})
```

Please use a more generic file name  (hopefully, we will have more PSI tests in future):
```suggestion
  add_rostest_gtest(planning_scene_interface_test planning_scene_interface.test planning_scene_interface.cpp)
  target_link_libraries(planning_scene_interface_test moveit_move_group_interface ${catkin_LIBRARIES})
```

---

so you would want to include all the tests in the same file right? I wasn't sure hence the name.

---

Another thing before I let out another commit, I'm not sure how to use the clang-format and the clang-tidy. Can you let me know the command of it.

Thanks : ) hopefully I wont make more silly mistakes like this.

---

> I'm not sure how to use the clang-format and the clang-tidy. Can you let me know the command of it.

See here: 
- https://moveit.ros.org/documentation/contributing/code/#pre-commit-formatting-checks
- https://moveit.ros.org/documentation/contributing/code/#clang-format-auto-code-formatting
- https://moveit.ros.org/documentation/contributing/code/#clang-tidy-linting